lib/db.js

```diff
@@ -133,6 +133,11 @@ var Db = module.exports = function(dbname, server) {
           callback(null, removed);
         });
       });
+      if(typeof callback!=='function') {
+        return new Promise(function(resolve){
+          callback = function(e,r){ resolve(r) };
```

Comment:
I don't think we want to lose the error if it happens. Need to check `if (e) reject(e) else resolve(r)` or something like that.